bin/test.sh

```diff
@@ -9,7 +9,7 @@ while getopts ":dp" opt; do
           docker-compose -f development-docker-compose.yml run --entrypoint /code/bin/test-entrypoint.sh api_development -p 8000
           ;;
         p)
-          docker-compose -f production-docker-compose.yml run --entrypoint /code/bin/test-entrypoint.sh "$DOCKER_SERVICE" -p 8000
+          docker-compose -f production-docker-compose.yml run --entrypoint /code/bin/test-entrypoint.sh $DOCKER_IMAGE -p 8000
```

Comment:
can someone inform me on if this change to $DOCKER_IMAGE from "$DOCKER_SERVICE" is a keeper or not.
@MikeTheCanuck @nam20485